### ISSUE TITLE
skipping https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm - transfer failed

### ISSUE THREAD

**Issue by metal3d** [dustymabe/vagrant-sshfs#126]
Hi, I cannot use sshfs with RockyLinux  guest:

```
The following SSH command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!

rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm

Stdout from the command:

Retrieving https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm


Stderr from the command:

curl: (22) The requested URL returned error: 403 Forbidden
error: skipping https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm - transfer failed
```

Why to not use the simple command `yum install epel-release` ?

**Comment by metal3d** [comment]
OK, for RockyLinux I had to change the command to:

```
dnf config-manager --set-enabled powertools 
```

in `~/.vagrant.d/gems/3.0.1/gems/vagrant-sshfs-1.3.6/lib/vagrant-sshfs/cap/guest/redhat/sshfs_client.rb`

**Comment by tuupola** [comment]
@metal3d What is the line you changed? I encountered the same problem.

**Comment by evenreven** [comment]
I have this issue as well. I solved it by disabling sshfs in the Vagrantfile, then vagrant ssh and `sudo dnf config-manager --set-enabled powertools` and then re-enabling sshfs. 

Maybe a dumb question, but why does it try to install epel6 in the first place? Does the plugin identify Rocky Linux 8 as RHEL6? It correctly identifies CentOS8, so it seems something is off somewhere.

**Comment by tuupola** [comment]
Rocky Linux guest support was added in [Vagrant 2.2.17](https://github.com/hashicorp/vagrant/blob/main/CHANGELOG.md#2217-july-7-2021). What is still needed is [guest capabilities file](https://github.com/dustymabe/vagrant-sshfs/tree/main/lib/vagrant-sshfs/cap/guest) for vagrant-sshfs.

**Comment by evenreven** [comment]
Thanks for the pointer on this @tuupola . I will attempt a local fix in my project, and (if it works well) then maybe attempt a PR in January.

**Comment by evenreven** [comment]
I forked this and got it working. Created a PR for it now: #130.

Thanks again for the help, @tuupola. 

Hope you have a chance to look at it, @dustymabe.

**Comment by tuupola** [comment]
@evenreven Awesome! It was in my TODO list to create a PR for this but I was procrastinating and never got around to do it.

**Comment by dustymabe** [comment]
This should be fixed by #130 and e76db4d
